tests/generation/FileGlobs.py

```diff
@@ -91,8 +89,8 @@ def test_get_project_files_gitignore(self):
         with suppress_stdout():
             self.assertEqual(
                 sorted(get_project_files(
-                    self.log_printer, self.printer, os.getcwd())[0]),
-                sorted(files))
+                    self.log_printer, self.printer, os.getcwd(),
+                self.noninteractive)[0]), sorted(files))
```

Comment:
Add a separate test for non-interactive mode. Don't touch the old tests because they shouldn't really be affected.